example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -81,10 +81,7 @@ describe('The Ingest Granule failure workflow', () => {
       deleteFolder(config.bucket, testDataFolder),
       cleanupCollections(config.stackName, config.bucket, collectionsDir, testSuffix),
       cleanupProviders(config.stackName, config.bucket, providersDir, testSuffix),
-      granulesApiTestUtils.deleteGranule({
-        prefix: config.stackName,
-        granuleId: inputPayload.granules[0].granuleId
-      })
+      granuleModel.delete({ granuleId: inputPayload.granules[0].granuleId })
```

Comment:
Obviously there's a reason you made this change but shouldn't calling the DELETE endpoint trigger this exact call to the delete method on the model?

Reply:
Yes, but it never gets to the model delete call. It was failing here https://github.com/nasa/cumulus/pull/1129/files/cc1efc5a6bb168b9ed5fadbf74a3c65bef45bf47#diff-9e56ea61b54ec49491ed43ea4a36a3a7L136 . It gets an error because of 
 ```
 UnhandledPromiseRejectionWarning: MultipleValidationErrors: There were 2 validation errors:
* MissingRequiredParameter: Missing required key 'Bucket' in params
* MissingRequiredParameter: Missing required key 'Key' in params
```
So the question I guess, is what should the behavior be of a the delete api when the thing being deleted is bad?  I  suppose I could fix this by making sure those errors are handled and maybe writing a warning to the logs?

FYI: The granule files being tests have no bucket or key because they were never normalized

Reply:
Yeah it feels like we should still delete it? Why would we care if the schema is bad on a delete?

Reply:
And then are you saying the test granule files are wrong?